CK3toEU4/Source/EU4World/Province/EU4Province.cpp

```diff
@@ -42,6 +43,16 @@ void EU4::Province::initializeFromCK3Title(const std::shared_ptr<CK3::Title>& or
 	details.owner = tagCountry.first;
 	details.controller = tagCountry.first;
 
+	// check for manual CK3 province name
+	if (srcProvince->isRenamed() && !srcProvince->isManualNameClaimed())
+	{
+		auto name = locDegrader.degradeString(srcProvince->getDisplayName());
```

Comment:
Locdegrader might not do anything on - say - cyrillic inputs. Yo should try renaming provinces into chinese and see what happens in eu4, just to be on the safe side.